### ISSUE TITLE
Search form button text missing

### ISSUE THREAD

**Issue by dreamwhisper** [brandid/course-maker-pro#10]
**Describe the bug**
As seen on https://demo.coursemakerpro.thebrandid.com/404, there's no text or indication what the search form is or how to search. It also appears that screen reader text is missing.
